projects/plugins/protect/src/class-jetpack-protect.php

```diff
@@ -181,6 +183,7 @@ public function initial_state() {
 			'siteSuffix'        => ( new Status() )->get_site_suffix(),
 			'jetpackScan'       => My_Jetpack_Products::get_product( 'scan' ),
 			'productData'       => My_Jetpack_Products::get_product( 'protect' ),
+			'siteSuffix'        => ( new Status() )->get_site_suffix(),
```

Comment:
```suggestion
```
`siteSuffix` is already declared on `183` - probably just left over from a merge conflict.